ReactWindows/ReactNative/Views/DatePicker/ReactCalendarDatePickerManager.cs

```diff
@@ -0,0 +1,232 @@
+        public override string Name
+        {
+            get
+            {
+                return "CalendarDatePickerWindows";
+            }
+        }
+
+        /// <summary>
+        /// Sets the value of the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="date">The new value.</param>
+        [ReactProp("date")]
+        public void SetDate(CalendarDatePicker view, DateTime? date)
+        {
+            view.Date = date ?? null;
+        }
+
+
+        /// <summary>
+        /// Sets the maximum allowed date of the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="date">The value to set as maximum.</param>
+        [ReactProp("maxDate")]
+        public void SetMaxDate(CalendarDatePicker view, DateTime? date)
+        {
+            if (date.HasValue)
+            {
+                view.MaxDate = date.Value;
+            }
+        }
+
+        /// <summary>
+        /// Sets the minimum allowed date of the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="date">The value to set as minimum.</param>
+        [ReactProp("minDate")]
+        public void SetMinDate(CalendarDatePicker view, DateTime? date)
+        {
+            if (date.HasValue)
+            {
+                view.MinDate = date.Value;
+            }
+        }
+
+        /// <summary>
+        /// Sets the placeholder text for the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="placeholderText">The value of the placeholder.</param>
+        [ReactProp("placeholderText")]
+        public void SetPlaceholderText(CalendarDatePicker view, string placeholderText)
+        {
+            view.PlaceholderText = placeholderText;
+        }
+
+        /// <summary>
+        /// Sets the <see cref="Windows.Globalization.CalendarIdentifiers"/> value for the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="identifier">The identifier, which must be one of the values defined in <see cref="Windows.Globalization.CalendarIdentifiers"/></param>
+        [ReactProp("calendarIdentifier")]
+        public void SetCalendarIdentifier(CalendarDatePicker view, string identifier)
+        {
+            if (identifier.Length > 0)
+            {
+                view.CalendarIdentifier = identifier;
+            }
+        }
+
+        /// <summary>
+        /// Sets the first day of the week for the picker.
+        /// </summary>
+        /// <param name="view">The picker instance.</param>
+        /// <param name="day">The day.</param>
+        [ReactProp("firstDayOfWeek")]
+        public void SetFirstDayOfWeek(CalendarDatePicker view, int day)
+        {
+            view.FirstDayOfWeek = (Windows.Globalization.DayOfWeek)day;
```

Comment:
Could simplify and add `using Windows.Globalization;` above.

Reply:
It would conflict with `System.DayOfWeek`.